src/views/catalog/customize/utils.ts

```diff
@@ -82,6 +82,12 @@ export const extractParameterNameFromMetadataName = (template: V1Template): stri
   return virtualMachineObject?.metadata.name?.replace(/[${}"]+/g, '');
 };
 
+export const getTemplateNameParameterValue = (template: V1Template): string => {
+  const nameParameter = extractParameterNameFromMetadataName(template) || 'NAME';
```

Comment:
`extractParameterNameFromMetadataName(template) || 'NAME'` is repeating here and in another file in this PR,
I would consider moving the `|| 'NAME'` part to the return value of the `extractParameterNameFromMetadataName` function